src/encoded/schemas/treatment_agent.json

```diff
@@ -93,7 +93,8 @@
                 "percent",
                 "units",
                 "U/mL",
-                "MOI"
+                "MOI",
+                "pfu"
```

Comment:
I'd suggest to write PFU in capital letters, to be consistent with MOI (or both small?). Unless this is the way they are most commonly used (I'm not too familiar with these units).

Reply:
both capital, IMO.